scripts/globals/ability.lua

```diff
@@ -469,10 +476,14 @@ function checkForElevenRoll(caster)
     local effects = caster:getStatusEffects()
     for _,effect in ipairs(effects) do
         if (effect:getType() >= EFFECT_FIGHTERS_ROLL and
```

Comment:
can't do this since naturalists and runeists arent sequential with the rest (doing it this way lets like 200 abilities pass this check).  It should probably be like (>= FIGHTERS and <= TACTICIANS) or (>= NATURALISTS and <= RUNEISTS)

Reply:
and then you don't need the runeists roll block after